index.html

```diff
@@ -170,6 +170,33 @@ <h2 class="letsconnect2">LET'S CONNECT</h2>
     </li>
     </ul>
     </section>
-  
+
+    <div class="foot-back">
+    <footer> 
+      <h1 class="footer-h1">Contact me</h1>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <form
+    action="https://formspree.io/f/xrgdbdop"
+    method="POST"
+  >
+     <ul class="ul-form">
+      <li>
+      <input type="text" name="name" placeholder="Write your name here" required>
+    </li>
+    <li>
+      <input type="email" name="email" placeholder="name@domain.com" required>
+    </li> 
+      
+    
+   
+      <li>
+      <textarea cols="37" rows="6" name="message" placeholder="Write your message here" required></textarea>
+    </li>
+    </ul>
```

Comment:
- [ ] kindly add the HTML validation for your form's inputs to avoid bad data from the user

> name should be maxlength 30
> text area should be maxlength 500